.github/workflows/release.yaml

```diff
@@ -584,7 +584,7 @@ jobs:
           docker image push --all-tags "${PUBLIC_RELEASE_IMAGE_BASE}"
 
       - name: Push package to PyPi
-        if: ${{ success() && !cancelled() }}
+        if: ${{ (release_checks.outputs.is_rc == 'false') && success() && !cancelled() }}
```

Comment:
small detail but maybe better to add 
```
env:
    IS_RC: ${{ needs.release-checks.outputs.is_rc }}
```
in this job and then call 
```
env.IS_RC == 'false' &&
```
like we do elsewhere, I find it a bit more clear and less error prone

Reply:
much better indeed! thanks for the suggestion!

A typo was detected thanks to that btw